src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -9804,6 +9808,75 @@ HRESULT ProfToEEInterfaceImpl::ApplyMetaData(
     return hr;
 }
 
+// Add path to list of assemblies used by the runtime to probe assembly references
+// pAssemblyPath should be absolute path of assembly
```

Comment:
It will be bad if assembly path will be relative, but we do not check it in code - I've not found any existing function for it - and with cross-platform support check for it may be tricky.

Reply:
Done